documentation/source/newsfragments/feature.1266.rst

```diff
@@ -0,0 +1,14 @@
+The cocotb log configuration is now less intrusive, and only configures the root logger instance, ``logging.getLogger()`` (:pr:`1266`).
```

Comment:
What time of markup are you thinking? I'm deliberately not linking to the method, because what I'm trying to give is an expression that evaluates to an objects.

Reply:
Then its fine of course.